docs/Using-Fleet/configuration-files/README.md

```diff
@@ -264,15 +270,928 @@ spec:
+- Default value: none.
+- Config file format:
+  ```
+  integrations:
+    zendesk:
+      - url: "https://example.zendesk.com"
+        email: "user1@example.com"
+        api_token: "secret"
+        group_id: 1234
+  ```
+
+##### integrations.zendesk[].group_id
+
+The group ID to use to create tickets.
+
+- Required setting (integer).
+- Default value: none.
+- Config file format:
+  ```
+  integrations:
+    zendesk:
+      - url: "https://example.zendesk.com"
+        email: "user1@example.com"
+        api_token: "secret"
+        group_id: 1234
+  ```
+
+##### integrations.zendesk[].enable_failing_policies
+
+Whether the integration is configured to create Zendesk tickets for failing policies.
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  integrations:
+    zendesk:
+      - url: "https://example.zendesk.com"
+        email: "user1@example.com"
+        api_token: "secret"
+        group_id: 1234
+        enable_failing_policies: true
+  ```
+
+##### integrations.zendesk[].enable_software_vulnerabilities
+
+Whether the integration is configured to create Zendesk tickets for recent software vulnerabilities.
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  integrations:
+    zendesk:
+      - url: "https://example.zendesk.com"
+        email: "user1@example.com"
+        api_token: "secret"
+        group_id: 1234
+        enable_software_vulnerabilities: true
+  ```
+
+### Organization Information
+
+#### org_info.org_name
+
+The name of the organization.
+
+- Required setting (string).
+- Default value: none (provided during Fleet setup).
+- Config file format:
+  ```
+  org_info:
+  	org_name: Fleet
+  ```
+
+#### org_info.org_logo_url
+
+The URL of the logo of the organization.
+
+- Optional setting (string).
+- Default value: none (uses Fleet's logo).
+- Config file format:
+  ```
+  org_info:
+  	org_logo_url: https://example.com/logo.png
+  ```
+
+### Server Settings
+
+#### server_settings.server_url
+
+The base URL of the fleet server, including the scheme (e.g. "https://").
+
+- Required setting (string).
+- Default value: none (provided during Fleet setup).
+- Config file format:
+  ```
+  server_settings:
+    server_url: https://fleet.example.org:8080
+  ```
+
+#### server_settings.live_query_disabled
+
+If the live query feature is disabled or not.
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  server_settings:
+    live_query_disabled: true
+  ```
+
+#### server_settings.enable_analytics
+
+If sending usage analytics is enabled or not.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  server_settings:
+    enable_analytics: false
+  ```
+
+#### server_settings.debug_host_ids
+
+<!-- This section used to be named Debug host, this ensures links with the #debug-host hash still work -->
+<span id="debug-host" name="debug-host"></span>
```

Comment:
```suggestion
```

I think we can remove this backlink.

I searched the Fleet repo and there are no links to the old section.